database/layer/admin-persona.js

```diff
@@ -73,9 +73,31 @@ const insertPersonaByTenantKey = async (tenantKey, subClaim, userContext = '') =
   }
 }
 
+const getPersona = async (personaKey) => {
+  try {
+    return await usherDb('personas').select('*').where({ key: personaKey }).first();
```

Comment:
@k1ch  is the `.first()` function needed here? We are using the pk for the look up, right?

Reply:
It is not necessary, but I need to destructure the response since it will be an array before returning the persona.